git/week1/jsweek1.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>javascript1-week1</title>
+</head>
+
+<body>
+    <h1> Home work javascript1-week1</h1>
```

Comment:
Is it an idea to include the js file here? 🙂 Or if not, why do you need HTML at all?

Reply:
I am hoping to see my work on the template page console, that is why I have HTML file.

Reply:
Then you are missing `<script src="jsweek1.js"></script>`